### ISSUE TITLE
can't tab to password prompt text field

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#2444]
*Issue migrated from trac ticket [#2290](http://colloquy.info/project/ticket/2290)*

**type:** Defect | **component:** Colloquy (Mac) | **severity:** Normal | **priority:** Normal | **version:** 2.3 (Mac) | **status:** new | **keywords:** accessibility
___
#### 2010-09-21 19:54:19: incanus@… created the issue

When prompted for a password to join a chat room, when Full Keyboard Access is enabled in System Preferences, only the Cancel & Join buttons are able to be tabbed to. The password prompt text field cannot be tabbed to and must be clicked with the mouse in order to enter the password. This should be a simple NIB change.
